core/src/commonMain/kotlin/com/ustadmobile/core/contentjob/ContentPlugin.kt

```diff
@@ -0,0 +1,39 @@
+package com.ustadmobile.core.contentjob
+
+import com.ustadmobile.door.DoorUri
+import com.ustadmobile.lib.db.entities.*
+import org.kodein.di.DIAware
+
```

Comment:
Let's add a comment here as this is a key interface:


/**
Implementations of ContentPlugin manage how content is imported, downloaded, and uploaded.
*/